static_src/components/form/form_text.jsx

```diff
@@ -21,20 +14,15 @@ export default class FormText extends FormElement {
   }
 
   render() {
-    const classes = [];
-
-    if (this.props.inline) {
-      classes.push('form_text-inline');
-    }
-
-    if (!!this.error) {
-      classes.push('error');
-    }
+    const classes = classnames({
+      'form_text-inline': this.props.inline,
+      error: !!this.error
+    });
 
     // Spaces in label give a healthy space for inline forms
     const label = <label htmlFor={ this.key }> { this.props.label } </label>;
     return (
-      <div className={ this.styler(classes) }>
+      <div className={ classes }>
```

Comment:
Maybe just inline `classes` - no need for a tmp var is there?

Reply:
I prefer to have a variable vs inlining a multiline call to `classNames`, the former is more readable to me.